src/cli/commands/init.tsx

```diff
@@ -156,16 +156,17 @@ export const Init: React.FC<InitProps> = props => {
 const Header: React.FC = () => {
   return (
     <Box flexDirection="column">
-      <Box width={80} textWrap="wrap" marginBottom={4}>
-        <Color white>
-          RudderTyper is a tool for generating strongly-typed{' '}
+      <Box width={80} marginBottom={4}>
```

Comment:
```suggestion
      <Box width={80} marginBottom={4} flexDirection="column">
```

Reply:
Does this fix the UI issues you've been observing?